test/integration_test.erl

```diff
@@ -91,20 +91,22 @@ prefix(Socket) ->
     {ok, Timer} = gen_server:call(statman_graphite_pusher, get_timer),
     pusher_pid() ! {timeout, Timer, {push, 60000}},
 
+    ok = gen_server:call(statman_graphite_pusher,{reload_prefix, <<"newprefix">>),
```

Comment:
I was unable to run the tests because of a syntax error. There is a closing brace missing on this line.

Extending the existing test is problematic. What is the purpose of <<"myprefix">> in the new test, since it only occurs once and there are no comparisons against it? I think the prefix reloading should have it's own test where the prefix changes with an assertion before and after the change.